oxide-vpc/src/engine/dhcpv6.rs

```diff
@@ -34,13 +44,46 @@ pub fn setup(
     cfg: &VpcCfg,
     ft_limit: NonZeroU32,
 ) -> Result<(), OpteError> {
-    // This layer contains no actions if the client has not been configured with
-    // IPv6 support.
-    let ip_cfg = match cfg.ipv6_cfg() {
-        None => return Ok(()),
-        Some(c) => c,
-    };
+    match cfg.ipv6_cfg() {
+        None => drop_all_dhcpv6(pb, ft_limit),
+        Some(ip_cfg) => add_dhcpv6_rules(pb, cfg, ip_cfg, ft_limit),
+    }
+}
+
+const LAYER_NAME: &'static str = "dhcpv6";
 
+fn drop_all_dhcpv6(
+    pb: &mut PortBuilder,
+    ft_limit: NonZeroU32,
+) -> Result<(), OpteError> {
+    // Predicates identifying any traffic destined for a DHCPv6 server.
+    let predicates = vec![
+        // Destined for the server multicast IP address.
+        Predicate::InnerSrcIp6(vec![
```

Comment:
I think this should be for the dest port?

Reply:
Yeah, nice catch.